Allow "ignorecase_always" as a synonym for "ignore_case_always" to
match the documentation.

diff --exclude=debian -Naur perl-5.8.8.orig/lib/Getopt/Long.pm perl-5.8.8/lib/Getopt/Long.pm
--- perl-5.8.8.orig/lib/Getopt/Long.pm	2005-12-15 11:33:58.000000000 +1100
+++ perl-5.8.8/lib/Getopt/Long.pm	2006-02-25 01:17:28.000000000 +1100
@@ -1262,7 +1262,7 @@
 	elsif ( $try eq 'ignorecase' or $try eq 'ignore_case' ) {
 	    $ignorecase = $action;
 	}
-	elsif ( $try eq 'ignore_case_always' ) {
+	elsif ( $try eq 'ignorecase_always' or $try eq 'ignore_case_always' ) {
 	    $ignorecase = $action ? 2 : 0;
 	}
 	elsif ( $try eq 'bundling' ) {
